src/plugin/tasks/struct/dynamodb-dy-tool.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
we said to move this to `serverless/snippets/serverless-dynamodb/src/dynamodb/dy.js`